test/job.js

```diff
@@ -10,66 +10,45 @@ const delay = require("delay");
 const sinon = require("sinon");
 const { Job } = require("../dist/job");
 const { Agenda } = require("../dist");
+const getMongoCfg = require("./fixtures/mongo-connector");
 
-const mongoHost = process.env.MONGODB_HOST || "localhost";
-const mongoPort = process.env.MONGODB_PORT || "27017";
-const agendaDatabase = "agenda-test";
-const mongoCfg =
-  "mongodb://" + mongoHost + ":" + mongoPort + "/" + agendaDatabase;
+let mongoCfg;
 
 // Create agenda instances
 let agenda = null;
-let mongoDb = null;
 let mongoClient = null;
 
-const clearJobs = () => {
-  return mongoDb.collection("agendaJobs").deleteMany({});
-};
-
-// Slow timeouts for CI
-const jobTimeout = 500;
+// Slow timeouts for Travis
+const jobTimeout = process.env.TRAVIS ? 2500 : 500;
```

Comment:
Seems that travis got dropped since my original work on this ... tagging line for removal

Reply:
Yes, please remove travis. Instead you might want to use the standard `CI` env var.

```
if (process.env.CI)
```

Reply:
@koresar Github PR diffs can be a bit strange at times: that line was still showing in the PR, but had been removed when I pulled it locally. I've pushed a change to restore it to the flat `const jobTimeout = 50;`